src/main/java/mk/learning/fileshare/constants/HashmapConstants.java

```diff
@@ -0,0 +1,13 @@
+package mk.learning.fileshare.constants;
+
+import java.util.HashMap;
+
+public class HashmapConstants {
+	
+	public static HashMap<String,HashMap<String,String>> mapOfMaps= null;
```

Comment:
This mapOfMaps is equivalent to the ExcelMapData. the parser service must modify this variable. Download service must access and read data from this variable only.